lib/performance_platform/gateway/s3_ip_locations.rb

```diff
@@ -0,0 +1,28 @@
+require 'aws-sdk-s3'
+
+class PerformancePlatform::Gateway::S3IpLocations
+  def initialize
+    @s3 = Aws::S3::Client.new(config)
+  end
+
+  def fetch
+    bucket = ENV.fetch('S3_PUBLISHED_LOCATIONS_IPS_BUCKET')
+    key = ENV.fetch('S3_PUBLISHED_LOCATIONS_IPS_OBJECT_KEY')
+
+    resp = s3.get_object(bucket: bucket, key: key)
```

Comment:
Just style comments.... I would throw `bucket` and `key` in initializer and use `response` rather than `resp`. Or is `resp` a CA norm?